moveit_ros/moveit_servo/include/moveit_servo/servo_parameters.h

```diff
@@ -155,4 +155,31 @@ struct ServoParameters
   rcl_interfaces::msg::SetParametersResult setParametersCallback(std::vector<rclcpp::Parameter> parameters);
 };
 
+// Helper template for declaring and getting ros param
+// Must be declared here to ensure template class is built for required templates when included
+template <typename T>
+void declareOrGetParam(T& output_value, const std::string& param_name, const rclcpp::Node::SharedPtr& node,
+                       const rclcpp::Logger& logger)
```

Comment:
Why did you remove the default value parameter .?

Reply:
Oh sorry, not sure. I'll fix that.

Reply:
Okay, I think this is good.